spec/factories.rb

```diff
@@ -557,6 +557,12 @@
       withdrawn_at { Time.zone.now }
     end
 
+    trait :dbd do
+      status { :declined }
+      declined_by_default { true }
+      decline_by_default_days { 10 }
```

Comment:
We should maybe add offer and offered_at?

Reply:
amended to use the `with_offer` trait to make sure all prerequisites are satisfied.